recipes/cgat-scripts/meta.yaml

```diff
@@ -0,0 +1,207 @@
+
+package:
+  name: cgat-scripts
+  version: 0.2.6
```

Comment:
Can you please move the version to the top as a jinja variable, and use the non-hash URL of pypi below? See the snakemake recipe for an example.